build.rs

```diff
@@ -1,8 +1,13 @@
 use diesel::prelude::*;
 use diesel_migrations::run_pending_migrations;
 use std::env;
+use std::error::Error;
 
 fn main() {
+    if let Err(err) = load_git_commit() {
+        println!("cargo:warning=failed to load git commit: {}", err);
```

Comment:
Note that this warning is triggered when deploying to heroku. I forget the details, but I think the git repo is not available during the build stage. We can obtain the hash at runtime on heroku like we do in the [`site_metadata` endpoint](https://github.com/rust-lang/crates.io/blob/ec91dd3a2b55d22a374338f9cb8f6937c0a620c4/src/controllers/site_metadata.rs#L9).

Of course this heroku environment variable is not available during local testing, but when testing locally with sentry reporting an "unknown" commit is probably fine.